src/main/java/com/teamrocket/tms/controllers/UserController.java

```diff
@@ -61,6 +62,17 @@ public ResponseEntity<TaskDTO> getTaskById(@PathVariable Long userId, @PathVaria
         return ResponseEntity.ok(userService.getTaskById(userId, taskId));
     }
 
+    @GetMapping("/{userId}/tasks/filtered")
```

Comment:
Please pay attention and try as much as possible to avoid verbs when designing endpoints, as a rule of thumb.

When designing RESTful APIs, it's recommended to use resource nouns rather than verbs. This approach aligns with the REST architectural style, which emphasizes resource-based interactions.
We can leave it as it is for now.